clients/cpp-console/docs/RaspberryPi.md

```diff
@@ -46,8 +46,35 @@ There are many ways to do development on a Raspberry pi. It may be useful to tak
 
   ```sh
   cd scripts
-  sh ./buildArm32.sh
+  ./buildArm32Linux.sh
   ```
+  
+## Build directly on Linux arm32 with Microsoft Audio Stack
+
+Microsoft Audio Stack can be used to enhance the experience.
+
+Benefits include:
+* Software Audio Echo Cancellation
+* Beamforming
+* Noise Suppression
+* Automatic Gain Control
+* Dereverberation
+
+In order to use Microsoft Audio Stack you must specify in the config file the appropriate values. Below are the ones that we used on our pi:</br>
```

Comment:
pi? You mean Raspberry Pie?

Reply:
Raspberry Pi see https://www.raspberrypi.org/

Reply:
Spelled it out. Is is Pi though not Pie. Pi the number not the dessert :)